Admin/AbstractAdminExtension.php

```diff
@@ -186,4 +186,16 @@ public function configureActionButtons(AdminInterface $admin, $list, $action, $o
     {
         return $list;
     }
+
+//    TODO: replace this doc block to inheritdoc in NEXT_MAJOR
+//    /**
+//     * Setting to true will enable mosaic button for the admin screen.
+//     * Setting to false will hide mosaic button for the admin screen.
+//     *
+//     * @param AdminInterface $admin
+//     * @param bool $isShown
+//     */
```

Comment:
> Why did you comment the doc block?

I saw this example in some commits that will merge without notice. So I try to do it in the same way.